feature-flags/README.md

```diff
@@ -0,0 +1,224 @@
+# FxA feature-flagging
+
+* [What do you mean, "feature-flagging"?](#what-do-you-mean-feature-flagging)
+* [How can I see the current state for all feature flags?](#how-can-i-see-the-current-state-for-all-feature-flags)
+* [How do I configure a new feature flag?](#how-do-i-configure-a-new-feature-flag)
+* [How do I edit the configuration for an existing feature flag?](#how-do-i-edit-the-configuration-for-an-existing-feature-flag)
+* [What should I do if I made a mistake when configuring a feature flag?](#what-should-i-do-if-i-made-a-mistake-when-configuring-a-feature-flag)
+* [How do I change the validation rules for a feature flag?](#how-do-i-change-the-validation-rules-for-a-feature-flag)
+* [How do I use a feature flag in the content server?](#how-do-i-use-a-feature-flag-in-the-content-server)
+* [How do I use a feature flag in the auth server?](#how-do-i-use-a-feature-flag-in-the-auth-server)
+
+## What do you mean, "feature-flagging"?
+
+Historically,
+the FxA team has carried out development work
+on feature branches,
+only merging to `master`
+after a given feature has been completed
+and code-reviewed.
+Then,
+every two weeks,
+all of the merged features
+from the current iteration
+are bundled together as a "train",
+progressed through QA
+and eventually deployed to production.
+Occasionally there are also point releases
+to address specific issues
+but broadly speaking,
+this two-week cycle
+defines how quickly
+any given feature
+can make it to production.
+
+In the future,
+we'd like to move towards
+a process of continuous deployment,
+where features are pushed to production
+a matter of minutes
+after they're merged,
+hidden behind a feature flag
+that can be used to control rollout
+(or rollback).
+This approach also facilitates
+mainline development,
+where engineers merge their work directly to `master`
+in regular, smaller chunks
+and releases are cut from
+dedicated branches off `master`.
+There's a much longer discussion
+to be had around mainline devleopment,
+which is out of scope for this readme.
+
+Anyway,
```

Comment:
The history might be better off in its own section and a concise definition of feature flagging presented here.

Reply:
Fwiw, I'm less interested in objectively defining what feature-flagging is than I am in conveying the cultural context for our team, that led me to work on it.